apps/change-log.md

```diff
@@ -14,6 +14,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-0?-??
```

Comment:
I would say here we use the dates for the upcoming roll out of 7.80 to announce the deprecation

```suggestion
## 2023-02-16
```